app/services/coronavirus/pages/content_builder.rb

```diff
@@ -139,9 +139,9 @@ def search_terms_in_sub_sections(items)
     end
 
     def search_terms_in_timeline
-      return [] if github_data["timeline"].blank?
+      return [] if page.timeline_entries.blank?
```

Comment:
I think you can call `timeline_data` directly here.

Reply:
I did try this and it worked. I decided not to go with it because `timeline_data` has to go through its own data manipulation loop which seemed a bit heavy handed. I opted for `page.timeline_entries` as a proxy because it's used in the `timeline_data` method on line 95 so is maybe a bit closer to what we're actually checking for. It also saves calling `timeline_data` twice. I don't feel too strongly about it though so happy to change this if you still want? :)

Reply:
Would that be solved by memoizing `timeline_data`?

Reply:
excellent shout, done!

Reply:
@JonathanHallam Have you been able to test this change on integration?

Reply:
@leenagupte I have now :) pushed to integration, republished the landing page and checked out the content item. All looks as it should